### PR TITLE
fix: upgrade angular 9

### PR DESCRIPTION
**Pull request by code2nguyen**

### CODE REVIEW

File: libs/angular-components/vector-map/src/vector-map.component.ts [hunk 1]
```diff
@@ -49,9 +48,9 @@ export class VectorMapComponent implements OnInit, OnDestroy, OnChanges {
   @Output() viewChange = new EventEmitter<VectorMapView>();
 
   countries: VectorMapCountry[] = [];
-  data: Partial<Plotly.Data>[] = [];
-  layout: Partial<Plotly.Layout> = {};
-  config: Partial<Plotly.Config> = {};
+  data: any[] = [];
```
Comment (aifrim) [inline libs/angular-components/vector-map/src/vector-map.component.ts]: Why remove this?
I have added them for intelisense :(

File: package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "clean": "rimraf dist coverage junit.xml",
     "start": "ng serve",
     "build": "ng build",
+    "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
```
Comment (aifrim) [inline package.json]: Who does this help?
`ng b` does it when you run it.

Why do we need it as a post install?

File: libs/angular-components/vector-map/src/vector-map.component.ts (hunk 1 above)
Comment (code2nguyen) [reply libs/angular-components/vector-map/src/vector-map.component.ts]: to remove this warning
Capture d’écran 2020-02-18 à 10 16 34

File: package.json (hunk 2 above)
Comment (code2nguyen) [reply package.json]: To speed up ng build on CI...https://angular.io/guide/ivy#speeding-up-ngcc-compilation

**Pull request merged**